README.md

```diff
@@ -71,11 +75,11 @@ Some Inputs are required, while others are optional.
     ```
```

Comment:
I know this is out of the scope of this change, but shouldn't the "dciTags" variable in the example above be renamed to "componentTags" as well?

Reply:
Yes, should be the same, thanks!